tools/ci-build.sh

```diff
@@ -21,9 +21,13 @@ fi
 
 case "$ci_buildsys" in
     (Makefile)
-        ./configure --enable-shared --enable-static --enable-wine --cross-prefix=${ci_target}-
-        make
-        make test
+        case "$ci_target" in
+            (*mingw32*)
+                ./configure --enable-shared --enable-static --enable-wine --cross-prefix=${ci_target}-
+                make
+                make test
+                ;;
+        esac
```

Comment:
Rather do not start "useless" travis job at all.

Travis supports specifying that some job created by implicit rules can be exluded, see:
https://docs.travis-ci.com/user/build-matrix/#excluding-jobs

I think that adding following YAML fragment should exclude all jobs which are using specific compiler and specific env variable:
```yaml
jobs:
  exclude:
    - compiler: gcc
      env: ci_buildsys=Makefile
```

Reply:
According to the mentioned compiler: notes should that not be 

```
jobs:
  exclude:
    - compiler: x86_64-linux-gnu-gcc
      env: ci_buildsys=Makefile
```

Reply:
Of course! If you add both compilers (32 and 64bit) you need to put two items:
```yaml
exclude:
 - compiler: first
   env: excluded_env
 - compiler: second
   env: excluded_env
```